src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -364,20 +364,25 @@
     <!-- Hook the target that fetches the CFamily files into to the build process. 
          This needs to be set after the SDK targets have been imported.
     -->
-    <PrepareForBuildDependsOn>$(PrepareForBuildDependsOn);GetCFamilyFiles</PrepareForBuildDependsOn>
+    <CoreCompileDependsOn>$(CoreCompileDependsOn);GetCFamilyFiles</CoreCompileDependsOn>
```

Comment:
We used to be running this target very early in the build process. I changed it to be a bit later, but it probably doesn't make any difference.

The reason I tried changing it was that previously the target was being executed by VS at design time i.e. when VS triggered a design-time build in the background. I wanted to see if moving it later would stop VS from triggering it repeatedly (it doesn't, but see below).

Reply:
Comment is no longer relevant - I reverted this change.